tests/variables.tf

```diff
@@ -13,13 +13,13 @@ variable "build_id" {
   type    = "string"
 }
 
-variable "cloudwatch_agent_url_lx" {
-  default = "s3://amazoncloudwatch-agent/linux/amd64/latest/AmazonCloudWatchAgent.zip"
+variable "cloudwatch_agent_url_centos" {
```

Comment:
I'd prefer to keep this variable name generic, `lx`. It's up to the user to pass the url for centos/redhat (though I'd be surprised if it were actually a different rpm).

Reply:
oic, this is in the test and we're only using centos, nvm